libsql/internal/ws/websockets.go

```diff
@@ -115,7 +116,12 @@ func (r *execResponse) value(rowIdx int, colIdx int) (any, error) {
 	return nil, fmt.Errorf("unrecognized value type: %s", val["type"])
 }
 
-func (ws *websocketConn) exec(ctx context.Context, sql string, sqlParams params, wantRows bool) (*execResponse, error) {
+func (ws *websocketConn) exec(
+	ctx context.Context,
+	sql string,
+	sqlParams params,
+	wantRows bool,
+) (*execResponse, error) {
```

Comment:
Please avoid reformatting code that's not changed by the PR.

Reply:
got it